### PR TITLE
"Copy to clipboard" doesn't work #2819

### PR DESCRIPTION
**Pull request by David-DB88**
- [ ] **Breaking change?** (if so, please describe the impact and migration path for existing application instances)

**What changes did you make?** (Give an overview)
set a warning message on copyToClipboard if user use http

**Is there anything you'd like reviewers to focus on?**

**How Has This Been Tested?** (put an "x" (case-sensitive!) next to an item)

- [ ] No need to
- [x] Manually (please, describe, if necessary)
- [ ] Unit checks
- [ ] Integration checks
- [ ] Covered by existing automation

**Checklist** (put an "x" (case-sensitive!) next to all the items, otherwise the build will fail)
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation (e.g. **ENVIRONMENT VARIABLES**)
- [x] My changes generate no new warnings (e.g. Sonar is happy)
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [x] Any dependent changes have been merged

Check out [Contributing](https://github.com/provectus/kafka-ui/blob/master/CONTRIBUTING.md) and [Code of Conduct](https://github.com/provectus/kafka-ui/blob/master/CODE-OF-CONDUCT.md)

**A picture of a cute animal (not mandatory but encouraged)**

### CODE REVIEW

File: kafka-ui-react-app/src/lib/hooks/useDataSaver.ts [hunk 1]
```diff
@@ -14,6 +14,13 @@ const useDataSaver = (
         title: '',
         message: 'Copied successfully!',
       });
+    } else {
+      showAlert('custom', {
+        id: subject,
+        title: '',
+        message:
+          'Copying to clipboard is unavailable due to unsecured (non-HTTPS) connection!',
```
Comment (Haarolean) [inline kafka-ui-react-app/src/lib/hooks/useDataSaver.ts:22]:
```suggestion
          'Copying to clipboard is unavailable due to unsecured (non-HTTPS) connection',
```

**Review by Haarolean — APPROVED** [review]

File: kafka-ui-react-app/src/lib/hooks/useDataSaver.ts [hunk 2]
```diff
@@ -14,6 +14,13 @@ const useDataSaver = (
         title: '',
         message: 'Copied successfully!',
       });
+    } else {
+      showAlert('custom', {
```
Comment (workshur) [inline kafka-ui-react-app/src/lib/hooks/useDataSaver.ts:18]: I would suggest to use `warning` here

File: kafka-ui-react-app/src/lib/hooks/useDataSaver.ts (hunk 1 above)
Comment (David-DB88) [reply kafka-ui-react-app/src/lib/hooks/useDataSaver.ts]: ok

File: kafka-ui-react-app/src/lib/hooks/useDataSaver.ts (hunk 2 above)
Comment (David-DB88) [reply kafka-ui-react-app/src/lib/hooks/useDataSaver.ts:18]: ok

**Review by Haarolean — APPROVED** [review]

File: kafka-ui-react-app/src/lib/hooks/useDataSaver.ts [hunk 3]
```diff
@@ -14,6 +14,13 @@ const useDataSaver = (
         title: '',
         message: 'Copied successfully!',
       });
+    } else {
+      showAlert('warning', {
+        id: subject,
+        title: '',
```
Comment (workshur) [inline kafka-ui-react-app/src/lib/hooks/useDataSaver.ts:20]:
```suggestion
        title: 'Warning',
```

**Review by workshur — APPROVED** [review]
Please add title for the message

**Comment by David-DB88** [comment]
ок

**Review by Mgrdich — APPROVED** [review]

**Review by armenuikafka — CHANGES_REQUESTED** [review]
Not able to copy the message because of connection.

https://user-images.githubusercontent.com/104780608/221556354-2cec4ad8-574d-4f34-acc8-dcd446de00fa.mov

**Comment by David-DB88** [comment]
@armenuikafka yes i showing warning  message because the connection is non Https and not secure  and the copy functionality is disabled for http, so it's working fine

**Review by workshur — APPROVED** [review]

**Review by Haarolean — APPROVED** [review]

**Pull request merged**